package.json

```diff
@@ -13,15 +13,14 @@
     "start:prod": "run-s build serve",
     "serve": "serve -s build",
     "build-js": "react-scripts-ts build",
-    "build": "npm-run-all build-clean build-css build-js copy-nginx",
+    "build": "npm-run-all build-clean build-css build-js",
     "build-clean": "rimraf ./build",
-    "copy-nginx": "sed 's/\\$BACKEND_SERVICE_NAME/'\"$BACKEND_SERVICE_NAME\"'/' ./nginx.conf > ./build/nginx.conf",
-    "test": "CI=true react-scripts-ts test --env=jsdom",
-    "test:i": "react-scripts-ts test --env=jsdom",
+    "test": "react-scripts-ts test --env=jsdom",
     "eject": "react-scripts-ts eject",
     "doc": "npx styleguidist server",
     "build-doc": "styleguidist build",
-    "deploy-doc": "yarn build-doc && gh-pages -d styleguide"
+    "deploy-doc": "yarn build-doc && gh-pages -d styleguide",
+    "precommit": "CI=true npm-run-all test build"
```

Comment:
you can remove this line (23) and keep the `test` (19) + `test:i` (20) (i is for interactive)

Reply:
Feel free to perform this change and push to the build branch